docker-compose.yml

```
@@ -37,6 +40,22 @@ services:
       driver: json-file
       options:
         max-size: 10m
+  
+  postgres-server:
+    image: postgres:14.11
```

```suggestion
  meta-database:
    image: postgres
```
1. 服务命名尽量语义化
2. 镜像不用限定版本，直接用最新的